cl/alerts/management/commands/cl_send_recap_alerts.py

```diff
@@ -611,11 +619,22 @@ def query_and_send_alerts(
                 continue
             search_type = search_params.get("type", SEARCH_TYPES.RECAP)
             results_to_send = process_alert_hits(
-                r, results, parent_results, child_results, alert.pk, query_date
+                r,
+                results,
+                parent_results,
+                child_results,
+                alert.pk,
+                query_date.date(),
             )
             if not results_to_send:
                 continue
             alerts_sent.append(alert.pk)
+
+            # Override query n in the 'View Full Results' URL to
```

Comment:
This looks like an incomplete thought or typo: 

```suggestion
            # Override query in the 'View Full Results' URL to
```

Reply:
Thanks. Fixed.